LBHackney-IT/lbh-social-care
# always show cases if user has unrestricted permission

**What**  
Always show cases if user has unrestricted permission

**How to test it**
- Set the `.env` `AUTHORISED_UNRESTRICTED_GROUP`
- check a person restricted `1316`
- the cases should be visible
